arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -3683,6 +3693,13 @@
 			path = "Distance composite symbol";
 			sourceTree = "<group>";
 		};
+		F1034FA122D3F5DA00B021DA /* Create terrain surface from a local tile package */ = {
+			isa = PBXGroup;
+			children = (
+				F1034FA222D3F5DA00B021DA /* CreateTerrainSurfaceFromLocalTilePackage.storyboard */,
+				F1034FA522D3F5DA00B021DA /* CreateTerrainSurfaceFromLocalTilePackageViewController.swift */,
+			);
+			path = "Create terrain surface from a local tile package";
```

Comment:
There is now a problem with this file preventing the project from being opened:

![image](https://user-images.githubusercontent.com/2257493/61561759-95c87e00-aa24-11e9-965a-b532519d4da8.png)

It looks like this group is missing the following two lines:

```swift
				sourceTree = "<group>";
			};
```

Please add them, making sure the indentation is correct, and then verify that the project can be opened.

Reply:
Besides just checking manually, is there a method to ensure correct indentation similar to ensuring correct indentation for the ContentPList file?

Reply:
Yes. Just like with plist files, Xcode regenerates the project file when it is saved. It doesn't look like you can explicitly save the project, but you could make any kind of change that affects the project file and then undo that change. For example, rearranging an item in the Project Navigator. Drag an item to reorder it, wait for the project file to update (a few seconds) and then drag the item back. Does that make sense?